neo/Core/Witness.py

```diff
@@ -12,37 +7,67 @@ class Witness(SerializableMixin):
     InvocationScript = None
     VerificationScript = None
 
-    def __init__(self, invocation_script=None, verification_script=None):
+    def __init__(self, invocation_script=bytearray(), verification_script=bytearray()):
+        """
+        Create an instance.
+
+        Args:
+            invocation_script (bytearray): the invocation script
+            verification_script (bytearray): the verification script
+
+        Throws:
+            Exception: if verification_script is a string
+        """
         try:
             self.InvocationScript = binascii.unhexlify(invocation_script)
         except Exception as e:
             self.InvocationScript = invocation_script
 
         if type(verification_script) is str:
-            raise Exception("CAnnot be string")
+            raise Exception("Cannot be string")
 
         try:
             self.VerificationScript = binascii.unhexlify(verification_script)
         except Exception as e:
             self.VerificationScript = verification_script
 
     def Size(self):
-        return sys.getsizeof(self.InvocationScript) + sys.getsizeof(self.VerificationScript)
+        """
+        Get the total size in bytes of the object.
+
+        Returns:
+            int: size.
+        """
+        from neo.IO.Helper import Helper as IOHelper
```

Comment:
don't do inline imports. import at top of files.

Reply:
I'd like to but I can't in this case because it will create a 'circular import' error.
`Witness` imports `neo.IO.Helper`, which imports `Transaction` which tries to import `Witness`. 

A possible solution is to make `neo.IO.Helper` no longer a class, but make it a simple a module from which you call the methods after importing (`from neo.IO import Helper` -> `Helper.<method x>`. Otherwise I don't know how to solve it. This was the one with the way with least impact I think. Ideas are welcome

Reply:
I see. If possible it is typically better to resolve circular important at the root instead of workarounds. Don't have much time atm to dive into it. Both ok for me I guess, although preferred resolve circular.